kubeval/kubeval_test.go

```diff
@@ -147,14 +149,19 @@ func TestValidateInputsWithErrors(t *testing.T) {
 	var tests = []string{
 		"invalid.yaml",
 		"multi_invalid.yaml",
+		"list_invalid.yaml",
 	}
 	for _, test := range tests {
 		filePath, _ := filepath.Abs("../fixtures/" + test)
 		fileContents, _ := ioutil.ReadFile(filePath)
 		config := NewDefaultConfig()
 		config.FileName = test
 		results, _ := Validate(fileContents, config)
-		if len(results[0].Errors) == 0 {
+		errorCount := 0
```

Comment:
Not sure if this was intentional, but this was only checking the first result. My fixture yaml initially failed this test.